doc/MediaSigning.xml

```diff
@@ -0,0 +1,1343 @@
+        </author>
+        <revremark>First release</revremark>
+      </revision>
+    </revhistory>
+  </info>
+  <chapter xml:id="chapter_zhs_njq_bwb">
+    <title>Scope</title>
+    <para>This specification defines how media from ONVIF devices is signed to give a guarantee that
+      the media has not been manipulated since it was transmitted from the device. This
+      specification further defines how an ONVIF client should act at file export to preserve the
+      signing feature, and also how an ONVIF client should validate the authenticity of a signed
+      stream. Audio is outside of scope.</para>
+  </chapter>
+  <chapter xml:id="chapter_a3s_njq_bwb">
+    <title>Normative references</title>
+    <para>ITU-T Recommendation, H.264: Advanced video coding for generic audiovisual services</para>
+    <para role="reference">&lt;<link xlink:href="https://www.itu.int/rec/T-REC-H.264"
+        xmlns:xlink="http://www.w3.org/1999/xlink"/>&gt;</para>
+    <para>ITU-T Recommendation, H.265: High effiency video coding</para>
+    <para role="reference">&lt;<link xlink:href="https://www.itu.int/rec/T-REC-H.265"
+        xmlns:xlink="http://www.w3.org/1999/xlink"/>&gt;</para>
+    <para>IETF RFC 6234 - US Secure Hash Algorithms (SHA and SHA-based HMAC and HKDF)</para>
+    <para role="reference">&lt;<link xlink:href="http://tools.ietf.org/html/rfc6234"
+        xmlns:xlink="http://www.w3.org/1999/xlink"/>&gt;</para>
+    <para>IETF RFC 3447 Public-Key Cryptography Standards (PKCS) #1: RSA Cryptography Specifications
+      Version 2.1</para>
+    <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink"
+        xlink:href="https://tools.ietf.org/rfc/rfc3447.txt"/>&gt;</para>
+    <para>ITU-T Recommendation X.690 (2008) | ISO/IEC 8825-1:2008, Information technology – ASN.1
+      encoding rules: Specification of Basic Encoding Rules (BER),Canonical Encoding Rules (CER)and
+      Distinguished Encoding Rules (DER)</para>
+    <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink"
+        xlink:href="https://www.itu.int/rec/T-REC-X.690-200811-S"/>&gt;</para>
+    <para>ITU-T Recommendation X.660 (2011) | ISO/IEC 9834-1:2012, Information technology —
+      Procedures for the operation of object identifier registration authorities — Part 1: General
+      procedures and top arcs of the international object identifier tree</para>
+    <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink"
+        xlink:href="https://www.itu.int/rec/T-REC-X.660/en"/>&gt;</para>
+    <para> Alliance for Open Media | AV1 Bitstream &amp; Decoding Process Specification </para>
+    <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink"
+        xlink:href="https://aomedia.org/av1/specification/"/>&gt;</para>
+    <para>Global OID reference database</para>
+    <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink"
+        xlink:href="https://oidref.com/"/>&gt;</para>
```

Comment:
Where is the actual reference to this list? (If it's ok to use oidref.com as a reference.